src/app/index.html

```diff
@@ -0,0 +1,95 @@
+<!doctype html>
```

Comment:
In that case, we don't need this index file at all correct? Since our html files will be placed in the django templates directory?